### ISSUE TITLE
[READ BEFORE POSTING] Bot not working

### ISSUE THREAD

**Issue by Sbrown-WilliamsCrusher** [Novalys/PokemonGo-Bot-SimpleGUI#271]
Niantic has changed their api!

Meaning our bot will no longer work with any version. This is a known issue of ours. This is not something to be reported to Niantic. 

I will edit this main post as more information is found out. 

Thank you for your patience.

--Message to @Novalys--
Please label as Discussion

**Comment by Mongril** [comment]
Ok cool cool, does this mean it will never run again? or do you guys know how to fix the problem? :]

**Comment by Makozer** [comment]
I think it will be fixed soon :)

**Comment by ElCubano** [comment]
Found the bot few minutes ago, was wondering. :D

**Comment by saranshgupta7** [comment]
same here stuck on evolution and transferring loop..!!!

hope the developers release the solution soon

btw thanks for this bot...

**Comment by apple911** [comment]
this bot is amazing hope you can fix it ! :)

**Comment by BootNati0n** [comment]
Thanks for all the amazing work! :)

**Comment by kelbouz** [comment]
If you manage to fix it, i'll donate! GLORY TO THE LORDS OF THE BOT!:)

**Comment by dickkyhenriette** [comment]
to bad i had so much fun...
so its not my Compiler!!

**Comment by apple911** [comment]
same if you do manage do fix il donate a good amount for your work

**Comment by Makozer** [comment]
throwing in some donation, too :+1:

**Comment by kelbouz** [comment]
Wave of donation incoming....TADAM!:)

**Comment by BGBlind** [comment]
ill throw in donationa s well ./.... this bot is awesome

**Comment by almydus** [comment]
I hope this is fixable.. I really enjoyed the unban feature on this!

**Comment by Sbrown-WilliamsCrusher** [comment]
This is fixable but it will take lots of time and reworking of the code. Hang tight!

**Comment by apple911** [comment]
any estimate on how long it could possible take ?

**Comment by OTTORE** [comment]
Thank you for the updates on this, as other have said, I appreciate the hard work you guys have put into this. I will patiently await a fix and would love to donate to your efforts! thank you!

**Comment by devilesh** [comment]
on Rocket api someone has updated the hash i think it might be the solution for the issue to please check it out 

update hash 0.31.1
https://github.com/FeroxRev/Pokemon-Go-Rocket-API/pull/114/commits/5a57002ccd82a815da3e28d598864751df529bea

**Comment by devilesh** [comment]
im fine on coding but i have never used github before so ill learn some more and try to compile it with the new hash and see if it works out

**Comment by Sbrown-WilliamsCrusher** [comment]
Check edit 3 @devilesh and thank you for the thought

**Comment by juanjoseca** [comment]
Hey this an awesome bot. I hope is fixed soon. Something that happened to my account is that all the pokemon disappeared (even my fav) :(

**Comment by bdroky** [comment]
any update ?

**Comment by Sbrown-WilliamsCrusher** [comment]
Please check the discord channel for updates. Currently there are none (timestamp:(CST) 6:52pm)

**Comment by NorthGang** [comment]
If you'll fix this i'll make it rain in Donations. also can you make this bot not for free?

**Comment by ElCubano** [comment]
How everybody pretends to donate After fixing . If You want to help than Donate and stop such bullshit.

**Comment by A-Charvin** [comment]
Message from @Novalys : 
> First, all of the bots use something called PROTOS, this is in light words the language the Pokemon App speaks with the Pokemon Server. This was done firstly by a guy called Ferox, since then there is another project that works just in doing this (https://github.com/AeonLucid/POGOProtos) which is awesome since they put so much detail on this, now... since I in all honesty don't have the time required to decode those API calls I need to wait until those guys understand what really changed and start with the decoding process. If it is true that this time requests are encrypted then it will take more time (will happen eventually), then I can start working again in updating this bot. Not sure if everyone knows but I started this project not with the intention of having bleeding edge features (I can't compete with others like Necro's who have 20 devs working) my only intention was to create a very simple GUI bot which can be used by anyone without much configuration. So, in other words I would find another hobby for the week more likely weekend because this could take a while.

Everything will be fine hopefully. :+1:

**Comment by Mongril** [comment]
Wudup again, if you can get it to work I'll legit donate afew dollars. Also maybe make the bot a paid for program? 5$? 

Hope you can fix it soon, I'm really keen to start using it again, also does anyone know of a bot that is currently working?

**Comment by Markmanus** [comment]
It seems somebody cracked the new api, but as i see he is not willing to share with others https://youtu.be/WhryILX3NI0

**Comment by Mongril** [comment]
Ahh I see, I left a comment asking if he could help out and release it, hopefully eveything with the bot gets fixed soon and they figure out the api problem, is there a paypal donation link or is it only bitcoin? if the dudes get this to work ill definitely donate some of my YT savings :p

whos the main dev/s?

**Comment by dickkyhenriette** [comment]
whos the main dev/s?
there are none its a competition.

**Comment by Markmanus** [comment]
https://m.facebook.com/pokemesh/ says they reversed it.  Now they need to implement(im quite in a doubt tho)

**Comment by dickkyhenriette** [comment]
reversed  cool thats fast ...its 010010 for me but i am still learning and its fun to see how they dit it

**Comment by Markmanus** [comment]
Thats for the pokemap.  Tbh i am. Not sure what is true and what is not... So many liars who wants credits resolving the issues first.

**Comment by Mongril** [comment]
Has it been resolved?

**Comment by Mongril** [comment]
Na it doesnt, no bots do

**Comment by yupdon** [comment]
@Markmanus Don't believe everything you see lol - if we don't have a working bot - neither do they.

**Comment by Markmanus** [comment]
Did i say i beleive? :)

**Comment by saranshgupta7** [comment]
The pokemongodev team just found the Unkown 6 Chunk...!!!

**Comment by Mongril** [comment]
Where did you hear this from?

**Comment by saranshgupta7** [comment]
necrobot announcement

**Comment by Mongril** [comment]
Could you post a link or something?

**Comment by saranshgupta7** [comment]
https://m.reddit.com/r/pokemongodev
see here it is

**Comment by saranshgupta7** [comment]
@juanjoseca yup that's what i am saying but pretty excited for the extensive work the devs are working on the api changes...

**Comment by juanjoseca** [comment]
@saranshgupta7 agree :) all we have to do know is wait :+1:

**Comment by JuicyDrucey** [comment]
my pogo bot starts but it only elapse time nothing else it doesn't go near pokestops or find anything for me hopefully there is a fix cause I haven't even gotten to use this program yet an there a many of good things that I have heard about this.

**Comment by NoFootprintsGG** [comment]
@JuicyDrucey You don't read other comments before writing your own one, do you? Thought so...

**Comment by SWALK25** [comment]
@NoFootprintsGG , apparently they don't, lmao. I downloaded this bot today, found out it wasn't loading, so I came to this area, and read all of the comments, before I decided to post anything about it. Obviously they're oblivious. Usually if something's not working, there's a good chance it's not working for numerous people.

**Comment by JuicyDrucey** [comment]
@NoFootprintsGG I read all the comments and no one specified that theres doesn't do anything they just said they had other problems like transfer loop and I know everyone knows that the bot is down but I was only say that I hope there is a fix to it because id like to use it so don't pull me out an make yourself look cool and smart buddy...

**Comment by Mongril** [comment]
@JuicyDrucey Bro you dont read lol

**Comment by Manastis** [comment]
@JuicyDrucey soo u read all the comments but not the topic? Let me pick the sentence that u need to read 

"[READ BEFORE POSTING] Bot not working #271

Niantic has changed their api! http://pokemongo.nianticlabs.com/en/post/aug2016-update/

Meaning our bot (and MANY others) will no longer work with any version. This is a known issue of ours."

hope i could help u

**Comment by bilbol** [comment]
Will it ever work again tho?

**Comment by AndrewDevelopment** [comment]
@bilbol probably yes https://www.reddit.com/live/xdkgkncepvcq

**Comment by NightmarexGR** [comment]
They fixed the api problem https://twitter.com/RedditPokeGoDev

**Comment by yupdon** [comment]
@NightmarexGR - fake.

**Comment by AiVo7z** [comment]
Somehow ppl are obviously too dumb to read that official devs for the api or better said the bot, already made all of their socialmedia and links public and said everything else is fake. that guy is an Attention whore who either got nothing or tries to get you some mailware shit.

However, i could bite my thumb off I only could use the bot 1 day before they changed the dam API but i gotta admit the bot worked ridicously good. Something like 10h running in Background already lvl 20-21. Evolving was nice as well, rly timesaving except the stupid evolves but hey couldn't care less the bot got enough resources and xp anyway.

Rly looking Forward to this and even tho this has been said tons of times I rly appreciate the time you put in and how great the bot was, thank for that!

**Comment by NightmarexGR** [comment]
seems it was fake true :(

**Comment by casaanova** [comment]
https://github.com/keyphact/pgoapi How about this one. Seems to be done. https://www.reddit.com/live/xdkgkncepvcq

**Comment by yupdon** [comment]
@casaanova Yes this is the right repo, they are nearing completion. See the repo's comments and commits for details.

**Comment by yupdon** [comment]
I'm assuming a few more commits and it will be ready to clone and implement here - which I'm sure the devs who have contributed to this bot and myself are eager to do. - I'll be available later this evening to start/ help / finish this.

**Comment by Sbrown-WilliamsCrusher** [comment]
Everyone please see Edit #.5 for the newest update

**Comment by yupdon** [comment]
Anyone getting the https://github.com/keyphact/pgoapi API converted to https://github.com/FeroxRev/Pokemon-Go-Rocket-API.git ?? - then let me know.

**Comment by cucurettu** [comment]
So now is possible to use the bot?

**Comment by bilbol** [comment]
They fixed the problem. But the bot has to be updated first

**Comment by minifreddusch** [comment]
@cucurettu The bot can't be used yet, the API has been uncovered, but it now has to be implemented into the bot before it can be used again.

**Comment by ericseb** [comment]
Link to discordchanel pls ty

**Comment by AiVo7z** [comment]
Pokefarmer works perfectly, just like before the API Change. I really don't like that bot tho, looking Forward to this one. It is possible guys, already botting atm.

**Comment by mmalikahmed** [comment]
Necrobot is up. But it does not have any UI so it make things complicated :) 
I am also waiting for this great bot

**Comment by devilesh** [comment]
https://discordapp.com/channels/208079791281143808/208079791281143808

**Comment by KurtKoko** [comment]
@mmalikahmed 
Necrobot has an 3rd Party GUI. It works well for me.
Just set up `"UseWebSocket": true` in Necrobot´s config.json and start the GUI. It has not the same functions as Novalys PoGoBot and you cannot edit your config from GUI, but better than nothing ;)

Found here

> 
https://github.com/nicoschmitt/necrobotvisualizer

**Comment by bigjustin** [comment]
so is this bot still down or is there a new download  i  cant seem to find anything ?

**Comment by minifreddusch** [comment]
Still down, use Necrobot.io instead.
